platform/podjs/src/browserPod.ts

```diff
@@ -353,83 +365,42 @@ export class IDBPolyOut implements PolyOut {
         throw "Not implemented: writeFile";
     }
 
-    /// destUrl should be the same as zipId
     async importArchive(url: string, destUrl?: string): Promise<string> {
-        const { fileName } = FileUrl.fromUrl(url);
-        const blob = await (await fetch(url)).blob();
+        const { data: dataUrl, fileName } = FileUrl.fromUrl(url);
+        const blob = await (await fetch(dataUrl)).blob();
         const db = await openDatabase();
 
-        if (destUrl) {
-            const zipId = destUrl;
-
-            return new Promise((resolve, reject) => {
-                const tx = db.transaction([OBJECT_STORE_POLY_OUT], "readonly");
-                const request = tx
-                    .objectStore(OBJECT_STORE_POLY_OUT)
-                    .get(zipId);
-
-                request.onsuccess = () => {
-                    let obj: FileInfo[] = [];
-                    if (request.result) {
-                        //request.result is readonly, so I make a shallow copy
-                        obj = [...request.result];
-                        obj.push({
-                            id: zipId,
-                            name: fileName,
-                            time: new Date(),
-                            blob: blob,
-                        });
-                    } else {
-                        obj = [
-                            {
-                                id: zipId,
-                                name: fileName,
-                                time: new Date(),
-                                blob: blob,
-                            },
-                        ];
-                    }
-
-                    const newTx = db.transaction(
-                        [OBJECT_STORE_POLY_OUT],
-                        "readwrite"
-                    );
-                    newTx.objectStore(OBJECT_STORE_POLY_OUT).put(obj, zipId);
-
-                    newTx.oncomplete = () => resolve(zipId);
-                    newTx.onerror = newTx.onabort = () => reject(newTx.error);
-                };
-
-                tx.onerror = tx.onabort = () => reject(tx.error);
-            });
-        }
-
         return new Promise((resolve, reject) => {
             const tx = db.transaction([OBJECT_STORE_POLY_OUT], "readwrite");
-            const zipId = "polypod://" + "FeatureFiles/" + createUUID();
-
-            tx.objectStore(OBJECT_STORE_POLY_OUT).put(
-                [
-                    {
-                        id: zipId,
-                        name: fileName,
-                        time: new Date(),
-                        blob: blob,
-                    },
-                ],
-                zipId
-            );
+            const id = destUrl || `polypod://${createUUID()}`;
```

Comment:
iOS and Android return `"polypod://" + "FeatureFiles/" + createUUID()` for the id. And pod.js does not. It would be nice to have it return the same.

Reply:
I see. The thing is you didn't adapt the regexp in `getFile` to account for that extra slash. So accessing any file failed. I suppose we can add `FeatureFiles/` back and adjust the regexp. But this PR got already merged, so feel free to submit a merge request if you like.

Reply:
I think we can leave it like this for now.